clang/lib/Sema/SemaSYCL.cpp

```diff
@@ -542,8 +542,7 @@ static void collectSYCLAttributes(SemaSYCL &S, FunctionDecl *FD,
   // In SYCL 1.2.1 mode, the attributes are propagated from the function they
```

Comment:
This comment about 1.2.1 should be dropped as well. I think that we should also do a pass over `AttrDocs.td` and drop 1.2.1 references from there as well, but that can be done as a separate `[NFC]` PR

Reply:
>>This comment about 1.2.1 should be dropped as well
done

Reply:
> This comment about 1.2.1 should be dropped as well.

done